### PR TITLE
Merge Prerelease beta/1.7.0 beta.2 back to main

### PR DESCRIPTION
**Pull request by PorterNan**
# What

Merge Prerelease beta/1.7.0 beta.2 back to main

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/communication-react/CHANGELOG.beta.md [hunk 1]
```diff
@@ -1,9 +1,46 @@
...
 
-This log was last generated on Tue, 27 Jun 2023 22:07:18 GMT and should not be manually modified.
+This log was last generated on Mon, 24 Jul 2023 17:34:36 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## [1.7.0-beta.2](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.7.0-beta.2)
+
+Mon, 24 Jul 2023 17:34:36 GMT 
+[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.7.0-beta.1...@azure/communication-react_v1.7.0-beta.2)
```
Comment (vhuseinova-msft) [inline packages/communication-react/CHANGELOG.beta.md:10]: Could you update 'Compare changes` link for 1.7.0 in the stable changelog?

**Review by edwardlee-msft — APPROVED** [review]

File: packages/communication-react/CHANGELOG.beta.md (hunk 1 above)
Comment (PorterNan) [reply packages/communication-react/CHANGELOG.beta.md:10]: This is beta change log - I guess we should keep comparing to its previous beta version?

Comment (vhuseinova-msft) [reply packages/communication-react/CHANGELOG.beta.md:10]: Yes, but we would need to update it in packages/communication-react/CHANGELOG.stable.md

Comment (PorterNan) [reply packages/communication-react/CHANGELOG.beta.md:10]: Interesting... For stable I am thinking that we should compare to previous stable release as well (but we are not doing that in our changelog), I will make the change but will raise the concern of this in our chat after this gets merged

**Review by vhuseinova-msft — APPROVED** [review]

**Pull request merged**